lib/octokit/client/repositories.rb

```diff
@@ -380,6 +380,39 @@ def repository_teams(repo, options = {})
       alias :repo_teams :repository_teams
       alias :teams :repository_teams
 
+      # List all topics for a repository
+      #
+      # Requires authenticated client for private repos.
+      #
+      # @param repo [Integer, String, Hash, Repository] A GitHub repository.
+      # @return [Array<Sawyer::Resource>] Hash of array representing topics.
```

Comment:
Do you mean "Array of hashes"?

Reply:
Aha, then `[Array<Sawyer::Resource>]` is what needs to change.

Reply:
I already changed in https://github.com/octokit/octokit.rb/pull/1044/commits/f927472fb879912d4dde80ece2ce194f762c2ce9, thank you.